controllers/video.js

```diff
@@ -21,6 +21,25 @@ exports.getVideoList = async (req, res, next) => {
   }
 };
 
+exports.getUserVideoList = async (req, res, next) => {
```

Comment:
user라는 느낌이 포괄적인 느낌이 들어서 마이페이지 느낌으로 MyVideo는 어떨까요?

```suggestion
exports.getMyVideoList = async (req, res, next) => {
```